lib/new_todo.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+
+class NewTodo extends StatefulWidget {
+  const NewTodo({super.key});
+
+  @override
+  State<NewTodo> createState() => _NewTodoState();
+}
+
+class _NewTodoState extends State<NewTodo> {
+  TextEditingController controller = TextEditingController();
+  List<String> allStrings = <String>[];
```

Comment:
What is allStrings used for? Looks like things are only added, but never read from here.